pkg/storage/gcs/gcs.go

```diff
@@ -18,6 +19,12 @@ import (
 	storageBase "github.com/openshift/managed-velero-operator/pkg/storage/base"
 	storageConstants "github.com/openshift/managed-velero-operator/pkg/storage/constants"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	veleroInstall "github.com/vmware-tanzu/velero/pkg/install"
```

Comment:
I'm not sure I like attaching the velero install into the storage driver here

Reply:
This is only needed for `veleroInstall.Deployment()` to create the initial object before tweaking it further and returning it.  Same in the `s3` driver.

A lot of platform-specific options go into the deployment object creation, but I'm open to other suggestions for abstracting that away from the main logic.